### ISSUE TITLE
Cannot install in new composer.json.

### ISSUE THREAD

**Issue by joe-hagerty** [matracine/ip-tools#2]
I have a brand new laravel website -

When I run:  `composer require mracine/ip-tools`

I get:

```
[InvalidArgumentException]
Could not fund a matching version of package mracine/ip-tools.  Check the pa
ckage spelling, your version constraint and that the package is available i
n a stability which matches your minimum-stability (dev)
```

It will install if I use matracine/ip-tools, so perhaps an issue with the namespaces?

**Comment by matracine** [comment]
Hello,
Sorry, I made a mistake in the composer install command.

use :
```
composer require matracine/ip-tools
```

**Comment by matracine** [comment]
Fixed: c50839c53e0eac636a81e763bf35ad4dca585d1a
